docs/scripts/prettify/prettify.js

```diff
@@ -0,0 +1,496 @@
+var q = null;
+window.PR_SHOULD_USE_CONTINUATION = !0;
```

Comment:
can you explain how is prettify used?

Reply:
The [JSDoc website](https://jsdoc.app/about-configuring-default-template.html) states that:
> By default, JSDoc's default template generates pretty-printed versions of your source files. It also links to these pretty-printed files in the documentation.

I believe that `prettify.js` is the file responsible for this. It looks that it has been replaced in the official JSDoc repo by the [Prettier](https://prettier.io) library, but active development of Docstrap probably ended before this change was made.

Reply:
i feel very uneasy to include source code of other third party software here, same with the icons, we should discuss them